environments/go/builder/build.sh

```diff
@@ -22,6 +22,15 @@ fi
 
 cd ${srcDir}
 
+# use private vendor is the vendor dir is exsit
+if  [ -d "vendor" ] && [ ! -z ${GOLANG_VERSION} ] && version_ge ${GOLANG_VERSION} "1.11"; then
+  if [ ! -f "go.mod" ]; then
```

Comment:
I think what following block doing is to prevent `cannot find module` issue.
```
if [ ! -f "go.mod" ]; then
     go mod init
     fi
```
Please add a line of comment to explain why we need it.